compose/compose-file/index.md

```diff
@@ -1209,13 +1209,20 @@ volumes:
     source: ./static
     target: /opt/app/static
 ```
+### restart
```

Comment:
@surajnarwade you need another return before the heading for the restart command or the markdown will show (as it is now on Netlify: https://deploy-preview-2912--docker-docs.netlify.com/compose/compose-file/#long-syntax-2). This only seems to be a problem on v3 (`index.md`) file. It seems to be done correctly on the v2 file.